[2024-10-01T13:43:09.443+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-01T13:43:09.454+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: smiles_to_s3.extract_data scheduled__2024-09-30T00:00:00+00:00 [queued]>
[2024-10-01T13:43:09.461+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: smiles_to_s3.extract_data scheduled__2024-09-30T00:00:00+00:00 [queued]>
[2024-10-01T13:43:09.462+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-10-01T13:43:09.474+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): extract_data> on 2024-09-30 00:00:00+00:00
[2024-10-01T13:43:09.489+0000] {standard_task_runner.py:57} INFO - Started process 200 to run task
[2024-10-01T13:43:09.503+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'smiles_to_s3', 'extract_data', 'scheduled__2024-09-30T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpguupr64e']
[2024-10-01T13:43:09.512+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask extract_data
[2024-10-01T13:43:09.552+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-01T13:43:09.603+0000] {task_command.py:410} INFO - Running <TaskInstance: smiles_to_s3.extract_data scheduled__2024-09-30T00:00:00+00:00 [running]> on host b4e1704a8a2b
[2024-10-01T13:43:09.615+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-01T13:43:09.683+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='smiles_to_s3' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-30T00:00:00+00:00'
[2024-10-01T13:43:09.716+0000] {python.py:183} INFO - Done. Returned value was: [{'smiles': 'CCO'}, {'smiles': 'C#N'}, {'smiles': 'CNN'}, {'smiles': 'CNO'}, {'smiles': 'CON'}, {'smiles': 'COO'}, {'smiles': 'CNNON'}]
[2024-10-01T13:43:09.743+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=smiles_to_s3, task_id=extract_data, execution_date=20240930T000000, start_date=20241001T134309, end_date=20241001T134309
[2024-10-01T13:43:09.773+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-10-01T13:43:09.801+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
